tools/mapping/gen_domain_files/src/gen_domain.F90

```diff
@@ -38,20 +38,26 @@ program fmain
   character(LEN=512) :: usercomment ! user comment
   character(LEN= 8)  :: cdate       ! wall clock date
   character(LEN=10)  :: ctime       ! wall clock time
+  real(r8) :: fminval = 0.001_r8    ! min allowable land fraction; frac set to 0 if frac < fminval
+  real(r8) :: fmaxval = 1.000_r8    ! max allowable land fraction; frac set to 1 if frac > fmaxval
```

Comment:
Do we have access to the global parameter `c1` here?

Reply:
`c1` is local to subroutine `gen_domain`, so no. Regardless, if we move this to the body of the program as you suggest above, I'd be inclined to set this as `fmaxval = 1` (no decimal or kind postfix) which will set `fmaxval` to the full precision allowed by the type `r8`. That is, I think using `c1` is unnecessary here.